css/client.less

```
@@ -7642,6 +7642,349 @@ body {
+
+.help-modal__button--primary {
+  margin-left: auto;
+  border: 0;
+  color: #fff;
+  background: linear-gradient(180deg, #b89167, #9a7c57);
+  box-shadow:
+    0 10px 22px rgba(80, 55, 25, 0.2),
+    inset 0 -2px 0 rgba(0, 0, 0, 0.14);
+}
+
+@keyframes helpBubblePulse {
```

Stylelint (17.12.0)

[error] Expected keyframe name "helpBubblePulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)